python/paddle/fluid/executor.py

```diff
@@ -397,12 +399,13 @@ def close(self):
             >>> ...
             >>> exe.close()
         """
-        if not self._closed and self.executor:
-            self.executor.close()
+        if not self._closed:
+            self._default_executor.close()
             self._closed = True
 
     def _run_parallel(self, program, scope, feed, fetch_list, fetch_var_name,
                       return_numpy):
+        exe = program._executor
```

Comment:
`program` is a bit confusing with `fluid.Program`, it should be `compiled_program`.
And the `exe` should be `parallel_exe`.

Reply:
not a big problem now, can improve in the future.